packages/cfd/src/Containers/mt5-compare-table-content.tsx

```diff
@@ -37,10 +37,15 @@ const Row = ({
     CFDs_restricted_countries,
     financial_restricted_countries,
     is_preappstore_restricted_cr_demo_account,
+    getResidence,
 }: TCompareAccountRowProps) => {
     const is_leverage_row = id === 'leverage';
     const is_platform_row = id === 'platform';
     const is_instruments_row = id === 'instruments';
+    const client_country = getResidence();
+    const is_af = client_country === 'af';
+    const is_mg = client_country === 'mg';
+    const is_pk = client_country === 'pk';
```

Comment:
I don't think it is the best approach here 😅 
Why do we need to have individual checks?

Reply:
Its a temporary fix as these countries are creating issue if we do a overall css class then they will effect the Indonesia and Singapore clients, thats why for time being these individual checks are placed. As we will be pushing revamp compare account.

Reply:
Removed it. Thanks